R/cdisc_data.R

```diff
@@ -33,7 +33,7 @@
 #'   )
 #' )
 cdisc_data <- function(...,
-                       join_keys = teal.data::join_keys(),
+                       join_keys = teal.data::cdisc_join_keys(...),
```

Comment:
At this moment `cdisc_join_keys()` doesn't work when `...` are old class types. When object in `...` is of class `TealDataset` or `TealDatasetConnector` or `TealDataConnector` function fails.

Previously `join_keys` default was just empty `JoinKeys` object which was refilled inside `cdisc_data` function body.

Reply:
Thanks for this, as we discussed in the call I've added this and refactored the code to remove repetitive code in `cdisc_data`